common/src/main/scala/ClassOneResult.scala

```diff
@@ -50,28 +53,80 @@ case class ClassOneRowOutput(
-        }(_.pure[Explained])
-      }
 
-      for {
-        a <- baseInterval
-        b <- if (periodQty == 1) a.pure[Explained] else {
-          (a * periodQty) gives s"$id: $a * $periodQty"
+      if (proRata)
+        moneyIntervalProRata(id)
+      else {
+        import Period._
+        val baseInterval = period match {
+          case Year => definition.year.pure[Explained]
+          case Month => definition.month.fold {
+            (definition.year / 12).gives(s"$id: year / 12 = ${definition.year} / 12")
+          }(_.pure[Explained])
+          case FourWeek => definition.fourWeek.fold {
+            (definition.year / 13).gives(s"$id: year / 13 = ${definition.year} / 13")
+          }(_.pure[Explained])
+          case Week => definition.week.fold {
+            (definition.year / 52).gives(s"$id: year / 52 = ${definition.year} / 52")
+          }(_.pure[Explained])
         }
-        rounded = b.mapBounds(_.roundNi)
-        c <- if (rounded == b) { b.pure[Explained] } else { rounded gives s"$id: ⌈$b⌉" }
-      } yield c
+
+        for {
+          a <- baseInterval
+          b <- if (periodQty == 1) a.pure[Explained] else {
+            (a * periodQty) gives s"$id: $a * $periodQty"
+          }
+          rounded = b.mapBounds(_.roundNi)
+          c <- if (rounded == b) {
+            b.pure[Explained]
+          } else {
+            rounded gives s"$id: ⌈$b⌉"
+          }
+        } yield c
+      }
+    }
+
+    private def moneyIntervalProRata(id: String): Explained[Interval[BigDecimal]] = {
+      def sequence(x: Bound[Explained[BigDecimal]]): Explained[Bound[BigDecimal]] = x match {
+        case EmptyBound() => (EmptyBound(): Bound[BigDecimal]).pure[Explained]
+        case Unbound() => (Unbound(): Bound[BigDecimal]).pure[Explained]
+        case Open(explained) => explained.map(Open(_))
+        case Closed(explained) => explained.map(Closed(_))
+      }
+
+      period match {
+        case Period.Year =>
+          definition.year.pure[Explained]
+
+        case Period.Week =>
+          val weekInterval: Explained[Interval[BigDecimal]] = definition.week.fold(
+            (definition.year / 52).gives(s"$id: year / 52 = ${definition.year} / 52")
+          )(_.pure[Explained])
+
+          weekInterval.flatMap{ w =>
+            def proRataBound(yearBound: Bound[BigDecimal], weekBound: Bound[BigDecimal], boundDescription: String): Explained[Bound[BigDecimal]] = {
+              val result = yearBound.map { y =>
+                val w: BigDecimal = weekBound match {
+                  case ValueBound(w) => w
+                  case _ => sys.error(s"Weekly $boundDescription bound value not found but annual $boundDescription bound value exists")
+                }
+
+                if ((y / 52).isWhole) (w * periodQty).gives(s"$id: $boundDescription yearBound / 52 = $y / 52 ∈ ℤ ⇒ $boundDescription bound = $periodQty * $boundDescription weekBound = $w")
```

Comment:
in the C44 documentation there are two different rules for calculating the pro-rata band limits. In one you multiply the weekly band limit by the number of weeks in the pro-rata period. In the other, you divide the annual band limit by 52, then multiply by the number of weeks in the pro-rata period and then round up to the nearest pound. I went through the band limits and found that the ones where you use the annual figure always corresponded to the cases where the annual figure didn't divide evenly by 52. The weekly ones seemed to be the ones where the annual figure divided evenly. That's the basis for this logic here.